subscription-service/src/test/kotlin/com/egm/stellio/subscription/config/ScheduledNotificationTest.kt

```diff
@@ -0,0 +1,241 @@
+package com.egm.stellio.subscription.config
+
+import com.egm.stellio.shared.model.Notification
+import com.egm.stellio.shared.util.JsonLdUtils
+import com.egm.stellio.shared.util.loadSampleData
+import com.egm.stellio.shared.util.toUri
+import com.egm.stellio.subscription.firebase.FCMService
+import com.egm.stellio.subscription.model.EntityInfo
+import com.egm.stellio.subscription.service.NotificationService
+import com.egm.stellio.subscription.service.SubscriptionEventService
+import com.egm.stellio.subscription.service.SubscriptionService
+import com.egm.stellio.subscription.utils.gimmeRawSubscription
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.client.WireMock
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration
+import com.ninjasquad.springmockk.MockkBean
+import io.mockk.*
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.annotation.Import
+import org.springframework.test.context.ActiveProfiles
+import reactor.core.publisher.Mono
+import reactor.kotlin.core.publisher.toMono
+import reactor.test.StepVerifier
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE, classes = [ScheduledNotification::class])
+@Import(WebClientConfig::class)
+@ActiveProfiles("test")
+class ScheduledNotificationTest {
+
+    @Autowired
+    private lateinit var scheduledNotification: ScheduledNotification
+
+    @MockkBean
+    private lateinit var notificationService: NotificationService
+
+    @MockkBean
+    private lateinit var subscriptionService: SubscriptionService
+
+    @MockkBean
+    private lateinit var subscriptionEventService: SubscriptionEventService
+
+    @MockkBean
+    private lateinit var fcmService: FCMService
```

Comment:
you are not using these 2 services in your job, they should not appear here.

Reply:
Ok I take them off